accounting/filters/ChtcScheddCpuRemovedFilter.py

```diff
@@ -72,7 +72,7 @@ def get_query(self, index, start_ts, end_ts, **kwargs):
                                 "JobStatus": 3
                             }},
                             {"regexp": {
-                                "ScheddName.keyword": "submit-?[0-9]+[.]chtc[.]wisc[.]edu"
+                                "ScheddName.keyword": "(submit-?[0-9]+|learn|ap[0-9]+)[.]chtc[.]wisc[.]edu"
```

Comment:
`learn` is hopefully a relic of the past, but it doesn't hurt to have it here as well.